sotodlib/toast/ops/sim_source.py

```diff
@@ -514,6 +514,10 @@ class SimSource(Operator):
         help="Initial distance of the artificial source in meters",
     )
 
+    source_azimuth_start = Quantity(
+        u.Quantity(0, u.degree), help="Start of the scan along the azimuth axis"
+    )
```

Comment:
This is technique (setting to zero and comparing to zero) is fine.  You can also set the trait to `None` as the default value and pass the `allow_none=True` option.  Then later in the code do `if self.source_azimuth_start is None: ...`.  But comparing a floating point number to zero should always work- just not comparisons to other values.